src/metabase/channel/params.clj

```diff
@@ -0,0 +1,27 @@
+(ns metabase.channel.params
+  (:require
+   [clojure.string :as str]
+   [metabase.driver.common.parameters :as params]
+   [metabase.driver.common.parameters.parse :as params.parse]))
```

Comment:
Ah nice we could just reuse this for the heavy lifting, pretty happy with this solution for now.